macos/FluentUI/Separator.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+@objc(MSFSeparatorOrientation)
+public enum SeparatorOrientation: Int {
+    case horizontal
+    case vertical
+}
+
+@objc(MSFSeparator)
+open class Separator: NSView {
+	let orientation: SeparatorOrientation
+	
+	/// Initializes a separator in the specified orientation
+	/// - Parameter orientation: The orientation of the separator, vertical or horizontal
+	@objc public init(orientation: SeparatorOrientation) {
+		self.orientation = orientation
+		super.init(frame: .zero)
+		wantsLayer = true
+		if let layer = layer {
+			layer.backgroundColor = separatorColor.cgColor
```

Comment:
oh wait you're right I'm wrong. We can't resolve the cgColor here because we're not in a known draw path so results may vary. Back to updateLayer it must go. Sorry for the bad advice

Reply:
Actually it's already in updateLayer. Just remove this and we're good